lib/faraday/adapter/em_http.rb

```diff
@@ -281,6 +281,8 @@ def check_finished
     warn 'Warning: no such file to load -- openssl. ' \
       'Make sure it is installed if you want HTTPS support'
   else
-    require 'faraday/adapter/em_http_ssl_patch'
+    if EventMachine::HttpRequest::VERSION < '1.1.6'
```

Comment:
Just require version. That's it.
```
require 'em-http-request/version'
if ....
```

Reply:
OK - that's what I had mentioned above. Just force-pushed an update that takes that approach